scripts/zones/QuBia_Arena/Globals.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: QuBia_Arena
+-- Globals
+-----------------------------------
+local ID = require("scripts/zones/QuBia_Arena/IDs")
+
+-----------------------------------
+-- Mission 9-2 SANDO
+-- BCNM: Heir to the light
+-----------------------------------
+function phaseChangeReady(battlefield)
```

Comment:
This creates a global accessable everywhere, this is a pattern we're trying to move away from. Instead you can do this:

```lua
-- QuBia_Arena/Globals.lua 
local global = {}

global.phaseChangeReady = function(battlefield)
end

return global


-- File you want to use it in
local global = require("scripts/zones/QuBia_Arena/Globals")

... global.phaseChangeReady(battlefield) ...
```

Reply:
Thanks it's done !